jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1010,6 +1011,43 @@ properties:
             type: number
             fits_keyword: YOFFSET
             blend_table: True
+          starting_set:
+            title: Starting 4-point set number
+            type: integer
+            enum: [1, 3, 5]
+            fits_keyword: DSETSTRT
+          number_of_sets:
+            title: Total number of 4-point sets
+            type: integer
+            enum: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+            fits_keyword: NUMDSETS
+          direction:
+            title: Direction of dither pattern offsets
+            type: string
+            # 2019-06-12: Remove the NULL value once the keyword dictionary has been corrected.
+            enum: [NEGATIVE, POSITIVE, "NULL"]
+            fits_keyword: DITHDIRC
+          optimized_for:
+            title: Dither pattern optimization
+            type: string
+            # 2019-06-12: Remove the underscore versions of these values once the keyword
+            # dictionary has been corrected.
+            # 2019-06-12: Remove the NULL value once the keyword dictionary has been corrected.
+            enum: ["POINT-SOURCE", "EXTENDED-SOURCE", POINT_SOURCE, EXTENDED_SOURCE, "NULL"]
```

Comment:
Theoretically SDP should always be populating with the hyphen versions of these values, not the underscore. But I guess it's good to have the coverage just in case.

Reply:
As a general rule, if we're not 100% sure what values PPS or SDP are going to be passing along, it's safer for us to just not use an enum list at all. Especially for these new dither-related keywords, for which the front-end systems (APT and PPS) are still in flux and hence these values may change frequently or this may not even be the correct list to start with. We can always come back and add an enum list when things have settled down more.

Reply:
I replaced this and other questionable enum constraints with a note.